tools.repos

```diff
@@ -3,3 +3,7 @@ repositories:
     type: git
     url: https://github.com/autowarefoundation/autoware_tools.git
     version: main
+  tools/external/autoware_auto_msgs:
```

Comment:
```suggestion
  tools/external/autoware_auto_msgs:
```

But we cannot do this either. The tools folder is occupied by the `autoware_tools.git` above.

This should go somewhere else.

Maybe new structure as:

`https://github.com/autowarefoundation/autoware_tools.git` goes to `tools/autoware_tools:`
`https://github.com/tier4/autoware_auto_msgs.git` goes to `tools/external/autoware_auto_msgs:`

But still, do we really need to add this old package back in autoware?